Resources/public/views/recommendations.html.twig

```diff
@@ -0,0 +1,31 @@
+<div id="recommendation-{{ recommendationId }}" class="ez-recommendation">
+    {{ 'Loading recommended items...'|trans }}
+</div>
+
+<script id="recommendation-template-{{ recommendationId }}" type="text/x-handlebars-template">
+    {{ template|raw }}
+</script>
+
+<script>
+    (function() {
+        var recoClient{{ recommendationId }} = new EzRecoRestClient({
+            scenarioId: '{{ scenarioId }}',
+            locationId: '{{ locationId }}',
+            limit: '{{ limit }}',
+            restUrl: '/recommendations/fetch' // TODO: dynamically replace with path from routing.yml
```

Comment:
No, this is a local rest API call (not a YC call) so it may looks as you want. I think you should be able to change it in some unexpected situations. At least... it would be nice to have such feature... but OK, it was only an idea, let's leave it as it is. There are more important things to do :-)